packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -224,23 +254,80 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
 
   bool _debugAssertChildListLocked() => childManager.debugAssertChildListLocked();
 
+  /// Verify that the child list index is in strictly increasing order.
+  ///
+  /// This has no effect in release builds..
+  void _debugVerifyChildOrder(){
```

Comment:
you only call this from asserts, so you could make it return `bool` instead (return null in release builds), and have more idiomatic call sites.